### ISSUE TITLE
Add allowed client version range as a configuration option

### ISSUE THREAD

**Issue by Jaren8r** [PaperMC/Velocity#602]
When running a network that only accepts the latest versions (or at the very least, not the full range of 1.8-1.18), it would be great if there was a configuration option that defined the client versions that are allowed to connect. If you connect with a version not allowed, it would disconnect you with a message showing "This server is only compatible with Minecraft versions 1.17.1-1.18" (or whatever version range you defined). This is already partially possible with a plugin, but it becomes a bit messy if you use modern forwarding because [Velocity drops connections from clients older than 1.13 before plugins get a chance to change the message](https://github.com/PaperMC/Velocity/blob/292b2f96ab8d65a9982dd787c626d1060e5472be/proxy/src/main/java/com/velocitypowered/proxy/connection/client/HandshakeSessionHandler.java#L139).

**Comment by Strahilchu** [comment]
Yeah I would like this as well so I can just set it to only allow specific version so people don't get the wrong idea and try to connect with client that the servers don't support
